src/sentry/static/sentry/app/views/organizationDetails/lightWeightInstallPromptBanner.jsx

```diff
@@ -0,0 +1,110 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {analytics} from 'app/utils/analytics';
+import {t} from 'app/locale';
+import Alert from 'app/components/alert';
+import SentryTypes from 'app/sentryTypes';
+import getPlatformName from 'app/utils/getPlatformName';
+import withConfig from 'app/utils/withConfig';
+import withUsersTeams from 'app/utils/withUsersTeams';
+import withTeamProjects from 'app/utils/withTeamProjects';
+import withApi from 'app/utils/withApi';
+
+class LightWeightInstallPromptBanner extends React.Component {
```

Comment:
We can avoid duplication here if we 1) refactor `InstallPromptBanner` to additionally accept a `projects` prop and 2) have `<LightWeightInstallPromptBanner>` whose render just looks something like:

```
render() {
    if (loading) { return null; }
    return <InstallPromptBanner {...props} />
}
```

...or use a prop like in `NoProjectMessage`